<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Employe</class>
 <widget class="QDialog" name="Employe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>481</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Yu Gothic UI Semibold</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter un employé</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Identifiant(login)</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Prénom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_id">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nom">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prenom">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_ajouter">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_adresse">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_telephone">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_cin">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>150</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Adresse</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>90</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Téléphone</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>150</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CIN(password)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nom_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>451</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Poste</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Admin</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Agent de reservation</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Agent Bus</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Agent Promotions</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Agent Trajets</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Agent Conventions</string>
      </property>
     </item>
    </widget>
    <zorder>label_24</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>lineEdit_id</zorder>
    <zorder>lineEdit_nom</zorder>
    <zorder>lineEdit_prenom</zorder>
    <zorder>pb_ajouter</zorder>
    <zorder>lineEdit_adresse</zorder>
    <zorder>lineEdit_telephone</zorder>
    <zorder>lineEdit_cin</zorder>
    <zorder>label_5</zorder>
    <zorder>label_6</zorder>
    <zorder>label_7</zorder>
    <zorder>lineEdit_nom_2</zorder>
    <zorder>label_15</zorder>
    <zorder>comboBox</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Afficher les employés</string>
    </attribute>
    <widget class="QTableView" name="tabemploye">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>391</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;okkk&lt;span style=&quot; color:#ff5500;&quot;&gt;jhg&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>631</width>
       <height>451</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pdf">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>410</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Imprimer</string>
     </property>
    </widget>
    <zorder>label_25</zorder>
    <zorder>tabemploye</zorder>
    <zorder>pdf</zorder>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Supprimer un employé</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_id_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>90</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Veuillez entrer l'identifiant de l'employé :</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>160</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>0</y>
       <width>761</width>
       <height>451</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label_23</zorder>
    <zorder>lineEdit_id_2</zorder>
    <zorder>label_4</zorder>
    <zorder>pb_supprimer</zorder>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Modifier un employé</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_id_3">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nom_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prenom_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_adresse_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_telephone_2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>90</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_cin_3">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>150</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Identifiant</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Prénom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Adresse</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>90</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Téléphone</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>150</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CIN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="update">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>100</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>0</y>
       <width>761</width>
       <height>451</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_poste">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>200</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>200</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Poste</string>
     </property>
    </widget>
    <zorder>label_22</zorder>
    <zorder>lineEdit_id_3</zorder>
    <zorder>lineEdit_nom_4</zorder>
    <zorder>lineEdit_prenom_2</zorder>
    <zorder>lineEdit_adresse_2</zorder>
    <zorder>lineEdit_telephone_2</zorder>
    <zorder>lineEdit_cin_3</zorder>
    <zorder>label_9</zorder>
    <zorder>label_10</zorder>
    <zorder>label_11</zorder>
    <zorder>label_12</zorder>
    <zorder>label_13</zorder>
    <zorder>label_14</zorder>
    <zorder>update</zorder>
    <zorder>lineEdit_poste</zorder>
    <zorder>label_16</zorder>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>Chercher un employé</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_cin_2">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="search">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>70</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Chercher</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>241</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic UI Semibold</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Veuillez entrer l'identifiant de l'employé :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>0</y>
       <width>761</width>
       <height>451</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTableView" name="tabemploye_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>601</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <zorder>label_21</zorder>
    <zorder>lineEdit_cin_2</zorder>
    <zorder>search</zorder>
    <zorder>label_8</zorder>
    <zorder>tabemploye_2</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
